app/viewmodels/helpers/ViewMovementHelper.scala

```diff
@@ -51,7 +53,8 @@ class ViewMovementHelper @Inject()(
       case Movement => constructMovementView(movementResponse)
       case Delivery => constructMovementDelivery(movementResponse)
       case Transport => viewMovementTransportHelper.constructMovementTransport(movementResponse)
-      case Items    => viewMovementItemsHelper.constructMovementItems(movementResponse)
+      case Items => viewMovementItemsHelper.constructMovementItems(movementResponse)
+      case Guarantor => constructMovementGuarantor(movementResponse)
```

Comment:
I don't know if we want to do this now or later, but I think moving this to a sub-helper like the `items` and `transport` ones above would help reduce the size of this class. It would also make the test specs smaller as those helpers could have theirown Unit Tests,